DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/docs/raml/account.raml

```diff
@@ -18,3 +18,11 @@
     description: Logout from the system
     responses:
       200:
+/mailvisibility:
+     get:
```

Comment:
is it full? where is the second description and return types for both and response code for first?